config.ru

```diff
@@ -1,5 +1,12 @@
 # This file is used by Rack-based servers to start the application.
 
+require 'rack'
+require './lib/foreman/telemetry_rack'
+require 'prometheus/middleware/exporter'
+
+use Foreman::TelemetryRack
```

Comment:
What about using this only when telemetry is enabled?

Reply:
Chicken and egg problem, the middleware must be used, but I've added a check in the middleware, if disabled no GC stat is called.